apps/backend/src/db/migrations/20260729103000_agent_outcome_indexes.sql

```
@@ -0,0 +1,12 @@
+-- Keyset scans for the cross-stream agent-outcomes read
+-- (GET /api/workspaces/:workspaceId/agent-outcomes). Each branch of the UNION
+-- orders on its own `occurs_at` column — `scheduled_for` for a follow-up (its
+-- firing time), `status_changed_at` for a delegation (its last transition) —
+-- so both need a workspace-scoped DESC index with `id` as the tiebreaker the
+-- cursor compares on.
+
+CREATE INDEX IF NOT EXISTS idx_agent_follow_ups_workspace_scheduled
+  ON agent_follow_ups (workspace_id, scheduled_for DESC, id DESC);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)